KIPs/kip-7-fungible_token.md

```diff
@@ -204,6 +277,73 @@ but interfaces and other contracts MUST NOT expect these values to be present.
 function burnFrom(address _from, uint256 _value) public 
 ```
 
+#### paused
+
+Returns `true` if the contract is paused or `false` if the contract is not paused.
+
+OPTIONAL - This method can be used to improve usability,
+but interfaces and other contracts MUST NOT expect these values to be present.
+
+```solidity
+function paused() public view returns (bool) 
+```
+
+#### pause
+
+Pauses all methods related with token transfer such as transfer, transferFrom, approve, increaseAllowance and decreaseAllowance methods of the contract. 
```

Comment:
```suggestion
Pauses all methods related with token transfer such as transfer, transferFrom and approve methods of the contract. 
```
Since there is no increaseAllowance and decreaseAllowance in KIP-7, I think it's a good idea to delete it here as well.

Reply:
The given list is an example. So I think all methods don't need to be described in this standard.
What do you think?

Reply:
increaseAllowance and decreaseAllowance are not defined here, so if you want to mention, I think you need a simple explanation separately about those.

Reply:
Ok. I think explaining them in this standard is useless. I will remove it.